### PR TITLE
Update some of the empty state copy

### PR DESCRIPTION
**Pull request by cicdw**
I've been poking around the UI and noticed some of these could use some minor adjustments.

### CODE REVIEW

File: src/utilities/savedFilters.ts [hunk 1]
```diff
@@ -56,12 +56,12 @@ export const oneWeekSavedSearch = new SavedSearch({
 
 export const excludeScheduledSavedSearch = new SavedSearch({
   id: null,
-  name: 'No scheduled',
+  name: 'Hide Scheduled Runs',
```
Comment (brandonreid) [inline src/utilities/savedFilters.ts:59]: I could be mistaken, but I believe we use sentence case for all the filters
```suggestion
  name: 'Hide scheduled runs',
```

**Review by brandonreid — APPROVED** [review]
One comment but looks a lot better. 💯

**Review by stackoverfloweth — APPROVED** [review]

File: src/components/NotificationsPageEmptyState.vue [hunk 2]
```diff
@@ -5,11 +5,11 @@
     </template>
 
     <template #heading>
-      Create a notification to get started
+      Create a notification to begin monitoring your flows
     </template>
 
     <template #description>
-      Send an alert when any flow enters any state you specify.
+      Send an alert when a flow enters a specific state.
```
Comment (zhen0) [inline src/components/NotificationsPageEmptyState.vue:12]: Send an alert when a run enters a specific state.

**Review by zhen0 — COMMENTED** [review]
One small suggestion. Thanks for the edits.

File: src/components/NotificationsPageEmptyState.vue (hunk 2 above)
Comment (cicdw) [reply src/components/NotificationsPageEmptyState.vue:12]: good call, will update

**Pull request merged**